### DISCUSSION TITLE
Multiple Projects, Diff-only GitInfo.g

### DISCUSSION THREAD

**Discussion by wahmedswl** [devlooped/GitInfo#170] [Q&A]
Hi,
Is there anyway to differentiate between changed only projects within GitInfo.g. Eg: Project1 and Project2 and changes are done in Project2, yet Project1 GitInfo.g is also same. This way it would be possible to verify changes happen in Project1 etc

UseCase: We currently use GitInfo information to create differential builds that only have changed files as we do manual copy of artifacts.
 
Thanks

**Comment by kzu** [comment]
I don't think GitInfo provides much value there. You can just run `git` commands from MSBuild (just like GitInfo does, btw) to get the precise behavior you want. Just peek at the [way GitInfo executes git](https://github.com/devlooped/GitInfo/blob/main/src/GitInfo/build/GitInfo.targets#L203-L212) commands and copy/extend in your own targets as needed, I'd say.
